client/app/components/Footer.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 
 import { react2angular } from 'react2angular';
 
-function Footer({ clientConfig, currentUser }) {
+export function Footer({ clientConfig, currentUser }) {
```

Comment:
I think we could make this the default export as well, in modules that have only one domain object to simplify the imports elsewhere, right?

Reply:
Until we drop the Angular support, the default export has to be `init` function. Although maybe we can create another register function for `.jsx` files, which will call `react2angular` on these components by itself. Anyway, out of scope for this PR.

Reply:
Sigh, I always forget that shim :)

Reply:
Hopefully soon, we will be able to forget it for good :D